<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QDialog" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Прохождение теста</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="testTitleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 16pt; font-weight: bold; color: #2c3e50; padding: 10px;</string>
     </property>
     <property name="text">
      <string>Название теста</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="progressLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 12pt; color: #7f8c8d; margin-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>Вопрос 1 из 10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="questionFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="questionTextLabel">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 12pt; padding: 10px; background-color: #ecf0f1; border-radius: 5px; color: black;</string>
        </property>
        <property name="text">
         <string>Текст вопроса</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="answerStackWidget">
        <property name="styleSheet">
         <string notr="true">padding: 10px;</string>
        </property>
        <widget class="QWidget" name="singleChoicePage">
         <layout class="QVBoxLayout" name="singleChoiceLayout"/>
        </widget>
        <widget class="QWidget" name="multipleChoicePage">
         <layout class="QVBoxLayout" name="multipleChoiceLayout"/>
        </widget>
        <widget class="QWidget" name="textAnswerPage">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="textAnswerField">
            <property name="styleSheet">
             <string notr="true">padding: 8px; font-size: 11pt;</string>
            </property>
            <property name="placeholderText">
             <string>Введите ваш ответ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 10px 15px;
    font-weight: bold;
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #2980b9;
}
QPushButton:disabled {
    background-color: #bdc3c7;
}</string>
       </property>
       <property name="text">
        <string>⬅️ Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 10px 15px;
    font-weight: bold;
    background-color: #27ae60;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #229954;
}</string>
       </property>
       <property name="text">
        <string>Далее ➡️</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 10px 15px;
    font-weight: bold;
    background-color: #e74c3c;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #c0392b;
}
QPushButton:disabled {
    background-color: #bdc3c7;
}</string>
       </property>
       <property name="text">
        <string>✅ Завершить тест</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
